File: src/main/java/com/github/xumeng/java/problems/easy/problem0344/Solution0344.java
package com.github.xumeng.java.problems.easy.problem0344;

public class Solution0344 {

    public void reverseString(char[] s) {
        if (s.length == 1) {
            return;
        }
        char temp;
        for (int i = 0, j = s.length - 1; i < j; i++, j--) {
            temp = s[i];
            s[i] = s[j];
            s[j] = temp;
        }
    }

}

File: src/main/kotlin/com/github/xumeng/kotlin/problems/easy/problem0344/Solution0344.kt
package com.github.xumeng.kotlin.problems.easy.problem0344

class Solution0344 {

    fun reverseString(s: CharArray) {
        if (s.size == 1) {
            return
        }
        var i = 0
        var j = s.size - 1
        var temp: Char
        while (i < j) {
            temp = s[i]
            s[i] = s[j]
            s[j] = temp
            i++
            j--
        }
    }

}